css/style.css

```diff
@@ -23,6 +23,7 @@ main {
 }
 
 #splash_screen {
+    height: 100%;
```

Comment:
For me, the splash screen is no longer the size of the game screen, double check if this is on your end too?

Reply:
Yeah, we decided as a group this made more sense as it allowed you to see the instructions at the beginning easier. Since there was no other content on the splash screen except a start button we didn't see it being an issue.

Reply:
ok cool!